### PR TITLE
Fix/ci2

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: capability-provider/Cross.toml [hunk 1]
```diff
@@ -12,8 +12,3 @@ image = "wasmcloud/cross:aarch64-apple-darwin"
 
 [target.x86_64-unknown-linux-gnu]
 image = "wasmcloud/cross:x86_64-unknown-linux-gnu"
-
-[build.env]
-passthrough = [
-    "XDG_CACHE_HOME",
```
Comment (connorsmith256) [inline capability-provider/Cross.toml:18]: I think this might still be needed?

Comment (autodidaddict) [reply capability-provider/Cross.toml:18]: I don't see it on other repos that work. But I'm just wildly guessing at this point

Comment (connorsmith256) [reply capability-provider/Cross.toml:18]: I see it on blobstore-fs, blobstore-s3, kv-vault, and direct-dispatch. I don't _not_ see it on any other provider

File: .ci/provider.yml [hunk 2]
```diff
@@ -72,13 +72,13 @@ stages:
...
             parameters:
-              workingDirectory: $(working-directory)     
+              workingDirectory: $(working-directory)
 
   - stage: release
     dependsOn:
       - build_and_check
     # Only do this stage if the ref is a tag and the previous stage succeeded
-    condition: and(succeeded('build_and_check'), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
+    #condition: and(succeeded('build_and_check'), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
```
Comment (connorsmith256) [inline .ci/provider.yml:81]: Just keeping this placeholder here so we don't actually merge it with this

**Review by connorsmith256 — CHANGES_REQUESTED** [review]

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**